packages/e2e/cypress/e2e/run/create-pipelinerun.cy.js

```diff
@@ -59,4 +64,77 @@ describe('Create Simple Pipeline Run', () => {
       .find('span[class="tkn--status-label"]', { timeout: 15000 })
       .should('have.text', 'Succeeded');
   });
+
+  it('should create pipelinerun yaml mode', function () {
+    const uniqueNumber = Date.now();
+
+    const pipelineRunName = `yaml-mode-${uniqueNumber}`;
+    const pipelineRun = `
+      apiVersion: tekton.dev/v1beta1
+      kind: PipelineRun
+      metadata:
+        name: ${pipelineRunName}
+        namespace: ${namespace}
+      spec:
+        pipelineSpec:
+          tasks:
+            - name: hello
+              taskSpec:
+                steps:
+                  - name: echo
+                    image: busybox
+                    script: |
+                      #!/bin/ash
+                      echo "Hello World!"
+    `;
+    cy.visit(`/#/pipelineruns/create`);
+
+    cy.contains('button', 'YAML Mode').click();
+    cy.url().should('include', 'mode=yaml');
+
+    cy.get('[class=cm-content]').type(pipelineRun);
```

Comment:
Can we provide an id or something for this element so we don't have to rely on implementation details of 3rd-party packages in the tests? If not, this is fine for now.

Reply:
Still requires something to get the correct div
```
<div id="create-pipelinerun--editor">
  <div>
    <div>
      <div
        aria-autocomplete="list"
        aria-multiline="true"
        autocapitalize="off"
        autocorrect="off"
        class="cm-content"
        contenteditable="true"
        data-language="yaml"
        role="textbox"
        spellcheck="false"
        style="tab-size: 4"
        translate="no"
      >
```

Reply:
OK let's stick with the class so

```suggestion
    cy.get('.cm-content').type(pipelineRun);
```